selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/WorkingWithSplitPanelTest.java

```diff
@@ -131,13 +131,7 @@ public void checkTerminalAndBuild() {
     multiSplitPanel.waitTabProcessIsPresent(1, "Terminal-2");
     terminal.waitTerminalIsNotEmpty(2); // terminal.waitTerminalIsNotEmpty(2)
     loader.waitOnClosed();
-    terminal.typeIntoActiveTerminal("mc");
-    terminal.typeIntoActiveTerminal(Keys.ENTER.toString());
-    terminal.waitTerminalIsNotEmpty(3);
-    loader.waitOnClosed();
-    String visibleTextFromTerminal = terminal.getVisibleTextFromTerminal(3);
-    stream(checkMcTerminal)
-        .forEach(content -> assertTrue(visibleTextFromTerminal.contains(content)));
+    typeIntoTerminalAndCheckTerminalIsNotEmpty();
```

Comment:
`checkTerminalAcceptsInputProperly()` method name is more relevant to what the method actually does, IMHO.